src/main/java/Duke.java

```diff
@@ -1,10 +1,59 @@
+import java.util.Scanner;  // Import the Scanner class
 public class Duke {
+
+    public static void drawLine() {  // Draw horizontal lines
+        for (int i = 0; i < 30; i++) {
+            System.out.print("_");
+        }
+        System.out.println();
+    }
+
     public static void main(String[] args) {
```

Comment:
The main method is very long >30 lines. Consider refactoring the inner parts of the code to different methods. [See here](https://nus-cs2113-ay2223s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods)